catalogue/webapp/test/e2e/helpers/utils.ts

```diff
@@ -0,0 +1,29 @@
+import { baseUrl } from './urls';
+
+export type cookieType = {
```

Comment:
```suggestion
export type CookieType = {
```

Reply:
> Note on ARIA labels:
> 
> I wonder if we should have them in 1 place to be able to audit them.
> 
> e.g.
> 
> ```
> `Radio checkbox ${workType.data.label}`
> ```
> 
> If that was visible somewhere as a label - we could have that audited and someone might say "it is bad practice to label it as a checkbox as the browser already does that". <= this is just an example, and made up.

its a good point. I could place all the aria labels into a file. Any dynamic variables can be replaced. e.g 'Radio checkbox %s'